README.md

```diff
@@ -122,6 +122,7 @@
 - [Nitro Knights](https://gamedistribution.com/games/nitro-knights) - Futuristic jousting game.
 - [Om Nom Run](https://play.famobi.com/om-nom-run) - Endless runner.
 - [Orbital Survival](https://playcanv.as/p/3G3RnfUz/) - Destroy wave after wave of enemies on a tiny world.
+- [Simply Up](https://simplyup.io) - Climb to the top of the tower in the shortest time.
```

Comment:
Shouldn't this be in the .IO section? While you can't interact with other players, you can at least see them?